src/backend/logger/logger.ts

```diff
@@ -32,17 +23,42 @@ export enum LogPrefix {
   WineTricks = 'Winetricks'
 }
 
-let longestPrefix = 0
+type LogInputType = unknown[] | unknown
+
+// helper to convert LogInputType to string
+function convertInputToString(param: LogInputType): string {
+  const getString = (value: LogInputType): string => {
+    switch (typeof value) {
+      case 'string':
+        return value
+      case 'object':
+        // Object.prototype.toString.call(value).includes('Error') will catch all
+        // Error types (Error, EvalError, SyntaxError, ...)
+        if (Object.prototype.toString.call(value).includes('Error')) {
+          return value!.toString()
```

Comment:
Checking for `value.stack` first and returning that (if it exists) might be more helpful

Reply:
I think `value.toString()`returns everything of Error. I will still check Error documentation.

Reply:
[`Error.stack`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack) *is* non-standard, but it gives more information than just calling `toString()` on it
Comparison:
```
$ node
Welcome to Node.js v16.16.0.
Type ".help" for more information.
> Error("Some Error").toString()
'Error: Some Error'
> Error("Some Error").stack
'Error: Some Error\n' +
  '    at REPL24:1:1\n' +
  '    at Script.runInThisContext (node:vm:129:12)\n' +
  '    at REPLServer.defaultEval (node:repl:566:29)\n' +
  '    at bound (node:domain:421:15)\n' +
  '    at REPLServer.runBound [as eval] (node:domain:432:12)\n' +
  '    at REPLServer.onLine (node:repl:893:10)\n' +
  '    at REPLServer.emit (node:events:539:35)\n' +
  '    at REPLServer.emit (node:domain:475:12)\n' +
  '    at REPLServer.Interface._onLine (node:readline:487:10)\n' +
  '    at REPLServer.Interface._line (node:readline:864:8)'
```
(in an actual program the stack would of course make more sense)